src/Framework/Profiler/EvaluationLocation.cs

```diff
@@ -37,6 +39,19 @@ public enum EvaluationPass : byte
         Targets
     }
 
+    /// <summary>
+    /// The kind of the evaluated location being tracked
+    /// </summary>
+    public enum EvaluationLocationKind : byte
```

Comment:
There's more kinds than these 3. Like properties, imports, etc. Now, all of the others will get defaulted to item, which is incorrect.

Ways to fix:
- exhaustively describe kind (look at all the usings and generate a kind for each location)
- the above but using broader terms (e.g. if property evaluation has 2 usings, name them both as Property)
- Introduce an Unspecified kind for anything that's not these current 3, and in the future we'll specify on demand. Should not break binary compat.

Reply:
Before this change we had just item and condition. I added glob, and that's why the enum.
I don't think we need more kinds at this point, so maybe this is just renaming 'item' to something else. Maybe 'element'?

---
In reply to: [156447130](https://github.com/Microsoft/msbuild/pull/2799#discussion_r156447130) [](ancestors = 156447130)